marlowe-playground-client/src/Wallet.purs

```diff
@@ -801,7 +801,7 @@ renderCurrentState state =
             <> tableRow
                 { title: "Accounts"
                 , emptyMessage: "No accounts have been used"
-                , columns: ("Account ID" /\ "Participant" /\ "Assets")
+                , columns: ("Participant" /\ "Assets" /\ mempty)
```

Comment:
We could use the opportunity to use one of the columns for the Token name

Reply:
Let's do it as a part of refining the UI/UX task.